### PR TITLE
Empty matrix

### PR DESCRIPTION
**Pull request by ThierryO**
Solves #70 

I took the liberty of moving the license file to a location where GitHub can find it. I'll revert that if you don't like it.

Updating the version number and ChangeLog is still to do.

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -18,6 +18,6 @@ Description: Implementation of a function 'digest()' for the creation
 URL: http://dirk.eddelbuettel.com/code/digest.html
 BugReports: https://github.com/eddelbuettel/digest/issues
 Depends: R (>= 2.4.1)
-License: GPL (>= 2)
+License: GPL (>= 2) | file LICENSE
```
Comment (eddelbuettel) [inline DESCRIPTION]: I don't think CRAN likes it that way.  I would revert that.

File: tests/digestTest.R [hunk 2]
```diff
@@ -165,7 +165,7 @@ for (i in seq(along.with=murmur32Input)) {
 }
 ## test 'length' parameter and file input
 ##fname <- file.path(R.home(),"COPYING")  ## not invariant across OSs
-fname <- system.file("GPL-2", package="digest")
+fname <- system.file("LICENSE", package="digest")
```
Comment (eddelbuettel) [inline tests/digestTest.R]: Same here.

File: tests/digestTest.R [hunk 3]
```diff
@@ -190,7 +190,7 @@ for (alg in c("sha1", "md5", "crc32", "sha256", "sha512",
 ## compare md5 algorithm to other tools
 library(tools)
 ##fname <- file.path(R.home(),"COPYING")  ## not invariant across OSs
-fname <- system.file("GPL-2", package="digest")
+fname <- system.file("LICENSE", package="digest")
```
Comment (eddelbuettel) [inline tests/digestTest.R]: And here.

**Comment by eddelbuettel** [comment]
So if it is to have the _repo_ have a license to display, I would simply make a copy, or a link.  GitHub's file system does not care as it has clever de-duplication.  We then add the new location to `.Rbuildignore` and all is well.

What I am a little more concerned about it the large set of lines in the output that changes.  What caused that?  This seems a little disruptive.

And over all this PR is not clear: what were you trying to do in the first place?  I feel the GPL-2 file move/renamer obfuscated something.  What issues with empty matrices needed fixing / were fixed?   In sum, this may be a good example of why starting with an issue ticket is better....

**Comment by eddelbuettel** [comment]
I see now re #70.  Absoliutely worth fixing, and the `storage.mode` change is good, as were the typos.

File: DESCRIPTION (hunk 1 above)
Comment (ThierryO) [reply DESCRIPTION]: It passes R CMD Check --as-cran. But I'll revert it.

**Comment by ThierryO** [comment]
Shall I bump the package version to 0.6.13.2?

**Comment by eddelbuettel** [comment]
Could do but don't have to.

The "why are there dozens of lines of digest output different" question is more important.  What drove that?

**Comment by ThierryO** [comment]
I think this is ready to merge now

**Comment by eddelbuettel** [comment]
(One totally minor thing I'll just fix on my end:  by some historic convention ChangeLog separates user name and email by two spaces. No need to update now.  I'll fix over morning coffee tomorrow.)

**Pull request merged**